targets/nuttx-stm32f4/README.md

```diff
@@ -85,12 +85,12 @@ make
 
 Connect Mini-USB for power supply and connect Micro-USB for `NSH` console.
 
-To configure `stlink` utility for flashing, follow the instructions [here](https://github.com/texane/stlink#build-from-sources).
+To configure `stlink` utility for flashing, follow the instructions in the official Stlink repository [here](https://github.com/texane/stlink).
```

Comment:
Consider this nitpicking, but why not put the link on "Stlink repository", why is "here" needed?